test/src/test/java/hudson/security/HudsonPrivateSecurityRealmTest.java

```diff
@@ -342,6 +346,41 @@ public void userCreationWithHashedPasswords() throws Exception {
         assertTrue(spySecurityListener.createdUsers.get(0).equals("charlie_hashed"));
     }
 
+    @Issue("JENKINS-56008")
+    @Test
+    public void updateUserPassword() throws Exception {
+        HudsonPrivateSecurityRealm securityRealm = new HudsonPrivateSecurityRealm(true, false, null);
+        j.jenkins.setSecurityRealm(securityRealm);
+        WebClient wc = j.createWebClient();
+
+        spySecurityListener.usersWithPasswordUpdate.clear();
+        assertTrue(spySecurityListener.usersWithPasswordUpdate.isEmpty());
+
+        // new user account creation
+        SignupPage signup = new SignupPage(wc.goTo("signup"));
+        signup.enterUsername("debbie");
+        signup.enterPassword("debbie");
+        signup.enterFullName(StringUtils.capitalize("debbie user"));
+        signup.enterEmail("debbie" + "@" + "debbie" + ".com");
```

Comment:
I like to use example.com as an example URL because it'll always be a sample domain. See http://example.com/

Reply:
Will add this in the next commit.